src/Services/Dominion/Actions/EspionageActionService.php

```diff
@@ -201,11 +201,10 @@ protected function performInfoGatheringOperation(Dominion $dominion, string $ope
                     ));
 
                 $spyLossSpaRatio = ($targetSpa / $selfSpa);
-                $spiesKilledPercentage = clamp($spiesKilledBasePercentage * $spyLossSpaRatio, 0.5, 1.5);
```

Comment:
We need to do something with this, giving an error on esp ops atm

Reply:
Why is it even used there?

Reply:
To prevent too small/large spy losses.

I suggest we bring it back and change the range to 0.1 - 1.5.

Min 0.1% losses (instead of 0.5%), max 1.5%?